queries/just/highlights.scm

```diff
@@ -0,0 +1,39 @@
+
+(assignment (NAME) @variable)
+(alias (NAME) @variable)
+(value (NAME) @variable)
+(parameter (NAME) @variable)
```

Comment:
```suggestion
(NAME) @variable
```

Reply:
I took a look at the parser today, and it seems `(NAME)` is used all over the place. It's basically a generic identifier match. Capturing all of them as `@variable` is too broad.